src/reactive/easyrsa.py

```diff
@@ -111,7 +112,7 @@ def configure_copy_extensions():
     """Update the EasyRSA configuration with the capacity to copy the exensions
     through to the resulting certificates."""
     # Create an absolute path to the file which will not be impacted by cwd.
-    openssl_file = os.path.join(easyrsa_directory, "openssl-1.0.cnf")
+    (openssl_file,) = glob.iglob(os.path.join(easyrsa_directory, "openssl-*.cnf"))
```

Comment:
This adjustment goes to support EasyRSA v3.0.9 rather than 3.0.1 this charm currently uses as its resource

Reply:
I've actually tested now with EasyRSA 3.1.7 and it works too